teachertool/src/utils/index.ts

```diff
@@ -11,4 +11,21 @@ export function makeNotification(
         duration,
         expiration: Date.now() + duration,
     };
+}
+
+export const isLocal = () => {
+    return window.location.hostname === "localhost";
+}
+
+// example: embedUrl for arcade is: https://arcade.makecode.com/
+export const getEditorUrl = (embedUrl: string) => {
```

Comment:
It just occurred to me, I believe this is built to essentially detect which target we're on and use the same target for the link we send to the iframe. This might end up being fine, but we also have the target in the shared project's metadata, so it may make more sense to construct the URL using that instead (see targetToUrl function in https://github.com/microsoft/pxt/compare/srietkerk-eval-iframe...thsparks/get_blocks_for_validation)

Not necessary for checking this in, but thought I'd mention it.

Reply:
Yeah, interesting. I think we'd need both of these functions for things to work as expected, but maybe not. Or at least a merging of their functionalities. If there are any trailing details to the url, if we just used the `targetToUrl` function, those details would get lost. The functionality in `getEditorUrl` is definitely important for skillmaps, to preserve which skillmap is being visited, so that's something to keep in mind. 

I won't address this in this PR, but I agree that it's something to discuss.